packages/create-ponder/tsup.config.ts

```diff
@@ -36,5 +37,18 @@ export default defineConfig({
             : name.replace(/^\./, "_dot_"),
       },
     );
+
+    readdirSync(targetPath)
```

Comment:
I think we'll also need to remove the `https://eth-sepolia.g.alchemy.com/v2/...` from the `.env.example` files in the `examples/` directory, right?